zhaquirks/sinope/light.py

```diff
@@ -29,29 +30,55 @@
     OUTPUT_CLUSTERS,
     PROFILE_ID,
 )
-from zhaquirks.sinope import SINOPE
+from zhaquirks.sinope import LIGHT_DEVICE_TRIGGERS, SINOPE
 
 SINOPE_MANUFACTURER_CLUSTER_ID = 0xFF01
 
 
 class SinopeTechnologiesManufacturerCluster(CustomCluster):
     """SinopeTechnologiesManufacturerCluster manufacturer cluster."""
 
+    class keypadLock(t.enum8):
+        """keypadLockout values."""
+
+        Unlocked = 0x00
+        Locked = 0x01
+
+    class action(t.enum8):
+        """actionReport values."""
+
+        single_on = 0x01
+        single_release_on = 0x02
+        long_on = 0x03
+        double_on = 0x04
+        single_off = 0x11
+        single_release_off = 0x12
+        long_off = 0x13
+        double_off = 0x14
```

Comment:
Zigpy uses a mix of `snake_case` and `CamelCase` for these. This should be: `Single_On`, `Single_Release_On`, ...

Although zigpy also isn't completely consistent in upper-casing the following letters. So I guess `Single_on` and so on would also be fine according to those conventions. (See [zigpy code as an example](https://github.com/zigpy/zigpy/blob/2cbb79f4335947a71c039a37ff26e87940dbd7a7/zigpy/zcl/clusters/hvac.py#L142-L161))